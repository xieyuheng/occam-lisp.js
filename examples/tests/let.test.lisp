(let ((id (lambda (x) x)))
  (id id))

(assert
  (equal?
   (let ((x 1)
         (y 1))
     (iadd x y))
   2))

(let ((x 1)
      (y 1))
  (assert
    (equal?
     (iadd x y)
     2)))
