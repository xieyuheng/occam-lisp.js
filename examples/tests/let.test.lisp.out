[apply] Can not apply lambda to zero arguments
  target: (lambda (x) (begin x))

